### PR TITLE
Set the wallet name reactive (fix #28)

### LINKED ISSUE #28: Wallet details: wallet names are not reactive
Wallet names should change in wallet details and wallet tiles as soon as updated

### PR DESCRIPTION
**Pull request by David0817**
fix #28

### CODE REVIEW

File: src/components/WalletSelectorField/WalletSelectorFieldTs.ts [hunk 1]
```diff
@@ -81,7 +83,8 @@ export class WalletSelectorFieldTs extends Vue {
     if (!this.knownWallets || !this.knownWallets.length) {
       return []
     }
-
+    // force update the knownWallets
+    this.hasUpdatedCurrentWallet
```
Comment (evias) [inline src/components/WalletSelectorField/WalletSelectorFieldTs.ts:87]: `this.hasUpdatedCurrentWallet` will just drop a boolean on the stack ; why did you add this line?

File: src/components/WalletSelectorPanel/WalletSelectorPanelTs.ts [hunk 2]
```diff
@@ -219,7 +220,8 @@ export class WalletSelectorPanelTs extends Vue {
     if (!this.knownWallets || !this.knownWallets.length) {
       return []
     }
-
+    // force update the knownWallets
+    this.hasUpdatedCurrentWallet
```
Comment (evias) [inline src/components/WalletSelectorPanel/WalletSelectorPanelTs.ts:224]: same here, useless.

File: src/store/Wallet.ts [hunk 3]
```diff
@@ -213,6 +214,11 @@ const walletState: WalletState = {
   transactionCache: {},
   // Subscriptions to websocket channels.
   subscriptions: {},
+  /**
+   * its own value what is true or false doesn't make sense,
+   * but a change in value means that the current wallet has been changed.
```
Comment (evias) [inline src/store/Wallet.ts:219]: change comment to: `A change in value means some of the current wallet values have changed.`

File: src/store/Wallet.ts [hunk 4]
```diff
@@ -392,6 +399,9 @@ export default {
       // - use Array.from to reset indexes
       return Vue.set(state, 'signedTransactions', Array.from(remaining))
     },
+    setHasUpdatedCurrentWallet:(state)=>{
```
Comment (evias) [inline src/store/Wallet.ts:402]: rename to `toggleHasUpdatedCurrentWallet`

File: src/components/WalletSelectorField/WalletSelectorFieldTs.ts (hunk 1 above)
Comment (vincent-lee90) [reply src/components/WalletSelectorField/WalletSelectorFieldTs.ts:87]: > `this.hasUpdatedCurrentWallet` will just drop a boolean on the stack ; why did you add this line?
`this.hasUpdatedCurrentWallet` is reactive, the   currentWallets will  re-evaluate when the `this.hasUpdatedCurrentWallet` changes
[https://vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods](url)

**Comment by David0817** [comment]
fix the conflict and remove useless defined

**Comment by evias** [comment]
WalletSelector**Panel** is not reactive anymore.

It works for WalletSelector**Field** but not for the Panel on the left side as shown in below screenshot. On the left side you can see the **old name** `evias-TAWJ` and on the top right bar you can see the **new name** `evias-TAWJ-1`. Please also make sure that the panel on the left gets updated.

Screenshot 2020-04-17 at 1 37 45 PM

**Comment by vincent-lee90** [comment]
> as per comment above

Now it is available.

**Comment by evias** [comment]
not to be merged ; fix to happen in #177
